src/Underworld.ts

```diff
@@ -4134,7 +4130,7 @@ function getEnemiesForAltitude(underworld: Underworld, levelIndex: number): stri
   // Prevent negative values which can happen during tutorial
   const adjustedLevelIndex = Math.max(0, levelIndex);
 
-  const numberOfTypesOfEnemies = 2 + Math.floor(adjustedLevelIndex / 2);
+  const numberOfTypesOfEnemies = Math.min(2 + Math.floor(adjustedLevelIndex / 2), 6);
```

Comment:
This would mean that the numberOfTypesOfEnemies never goes above 6.
What's the reasoning here?

Reply:
Beyond 6 enemy types means the levels lose their theme.

Now that I'm thinking about, I don't think this even comes into play until post deathmason. Happy to revert

Reply:
Sounds good to me, we can leave it in.